katello_certs_tools/katello_ssl_tool.py

```diff
@@ -147,14 +167,14 @@ def genPrivateCaKey(password, d, verbosity=0, forceYN=0):
             % ('%s', CRYPTO, repr(cleanupAbsPath(ca_key))))
 
     if verbosity >= 0:
-        print "Generating private CA key: %s" % ca_key
+        print("Generating private CA key: %s" % ca_key)
```

Comment:
please add `from __future__ import print_function` at the beginning of the file (if this does not need to be Python 2.4 compatible)